### PR TITLE
Hover bug fix & description in styles.md

### PR DESCRIPTION
**Pull request by Dariia**
### 🔦 Summary
...

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [x] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

File: src/SortableList/SortableList.driver.private.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const sortableListFactory = ({ element, wrapper }) => {
   return {
     ...publicSortableListDriver({ element, wrapper }),
     getDelayWrapper: () => element.querySelector('[data-hook="delay-wrapper"]'),
+    getItem: id => element.querySelector(`[data-hook="${id}"]`),
```
Comment (shlomitc) [inline src/SortableList/SortableList.driver.private.js]: this is redundant since the consumer is rendering the item on his own, so he has the data-hook for it.
just query the selector from test

Comment (Dariia) [reply src/SortableList/SortableList.driver.private.js]: @shlomitc please look on changes

File: src/SortableList/SortableList.js [hunk 2]
```diff
@@ -226,14 +226,23 @@ export default class SortableList extends WixComponent {
     );
   }
 
+  renderItem = args => {
+    const dropped = this.context.dragDropManager.monitor.didDrop();
+    const dragging = this.context.dragDropManager.monitor.isDragging();
+
+    return this.props.renderItem({
+      ...args,
+      smthDragging: dragging && !dropped,
```
Comment (shlomitc) [inline src/SortableList/SortableList.js]: please rename to `isListInDragState`

File: src/SortableList/SortableList.driver.private.js (hunk 1 above)
Comment (shlomitc) [reply src/SortableList/SortableList.driver.private.js]: `getElement` is better, but it just returns the native element.
It can also be dropped (you can just query the `wrapper` element), but it's not a biggie

Comment (Dariia) [reply src/SortableList/SortableList.driver.private.js]: ok

File: src/SortableList/SortableList.js (hunk 2 above)
Comment (Dariia) [reply src/SortableList/SortableList.js]: ok

File: stories/DragAndDrop/Styles/Styles.md [hunk 3]
```diff
@@ -20,8 +20,11 @@ Your component should use these styles and extend with any custom styles if need
  - `item` - defines a single item resets. should be applied to the root of your item
  - `itemPlaceholder` - defines how an item's placeholder (the empty section after dragging) looks like. should be applied to the root of your item in placeholder mode
  - `itemPreview` - defines how an item should look while it is dragged. should be applied to the root of your item in preview mode
-
-
+ - `isDragging` class - added to all items when any element is under dragging process and removed when process ended
```
Comment (Faradey27) [inline stories/DragAndDrop/Styles/Styles.md]: maybe isListInDragState?

File: src/SortableList/SortableList.js [hunk 4]
```diff
@@ -226,14 +226,23 @@ export default class SortableList extends WixComponent {
     );
   }
 
+  renderItem = args => {
+    const dropped = this.context.dragDropManager ? this.context.dragDropManager.monitor.didDrop() : true;
```
Comment (Faradey27) [inline src/SortableList/SortableList.js]: not sure that this is good solution

at least we need to know, why it can be undefined

File: src/MultiSelect/InputWithTags.js [hunk 5]
```diff
@@ -106,12 +107,14 @@ class InputWithTags extends React.Component {
         data-hook={this.props.dataHook}
       >
         {onReorder ? (
-          <SortableList
-            contentClassName={styles.tagsContainer}
-            items={tags}
-            onDrop={onReorder}
-            renderItem={this.renderReorderableTag}
-          />
+          <DragDropContextProvider>
```
Comment (shlomitc) [inline src/MultiSelect/InputWithTags.js]: `SortableList` already uses `DragDropContextProvider` internally, so why is this needed?
